clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/client/CloudFoundryApiException.java

```diff
@@ -54,4 +56,10 @@ private static String getMessage(String... errors) {
     return "Cloud Foundry API returned with error(s): " +
       stream(errors).filter(Objects::nonNull).collect(Collectors.joining(" and "));
   }
+
+  private static String getMessage(Throwable t, String... errors) {
+    String[] allErrors = Arrays.copyOf(errors,errors.length + 1);
```

Comment:
nit: missing space?

Reply:
thanks!

Reply:
The way that IJ puts the parameter names in line for you makes this difficult to see.